### PR TITLE
chore: bump versions and add topology spread constraints

### LINKED ISSUE #158: add topologySpreadConstraints to deployments
I would like to request that you add topologySpreadConstraints to the deployments so that I can specify minDomains. For reliability, we typically set a minimum of two AZs in our EKS clusters. This is a more efficient method than using pod anti-affinity and has been the recommended approach since k8s 1.25.

### PR DESCRIPTION
**Pull request by langchain-infra**
Closes #158

### CODE REVIEW

File: charts/langsmith/examples/read_only_config.yaml [hunk 1]
```diff
@@ -163,9 +163,9 @@ clickhouse:
   statefulSet:
     podSecurityContext:
       runAsNonRoot: true
-      runAsUser: 1000
-      runAsGroup: 1000
-      fsGroup: 1000
+      runAsUser: 101
```
Comment (bvs-langchain) [inline charts/langsmith/examples/read_only_config.yaml:166]: why this change?

**Review by bvs-langchain — APPROVED** [review]
lgtm, rerun helm-docs as well?

File: charts/langsmith/examples/read_only_config.yaml (hunk 1 above)
Comment (langchain-infra) [reply charts/langsmith/examples/read_only_config.yaml:166]: was incorrect, we use the right value in ci but not in the example
